test/runtests.jl

```diff
@@ -52,7 +52,7 @@ function get_logging_level(env_name::String, default)
 end
 
 function run_tests()
-    console_level = get_logging_level("SYS_CONSOLE_LOG_LEVEL", "Error")
+    console_level = get_logging_level("SYS_CONSOLE_LOG_LEVEL", "Info")
```

Comment:
It is best to change the logging level only when running the simulations tests. @daniel-thom is it possible to reset the logging level for the simulation tests?